operator/internal/controller/vectorized/test/cluster_controller_configuration_test.go

```diff
@@ -642,7 +755,8 @@ var _ = Describe("RedpandaCluster configuration controller", func() {
 
 			By("Having it reflected in the configuration")
 			Eventually(testAdminAPI.PropertyGetter(managedProp), timeout, interval).Should(Equal(desiredManagedPropValue))
-			Eventually(testAdminAPI.PropertyGetter(unmanagedProp), timeout, interval).Should(Equal(unmanagedPropValue))
+			// Unmanaged properties are removed by Declarative mode
+			Eventually(testAdminAPI.PropertyGetter(unmanagedProp), timeout, interval).Should(BeNil())
```

Comment:
This is the sole behavioural change moving from the ThreeWayMerge to the Declarative Syncer.